### ISSUE TITLE
Empty string supplied as input

### ISSUE THREAD

**Issue by tomsommer** [duncan3dc/sonos#91]
```
PHP Warning:  DOMDocument::loadXML(): Empty string supplied as input in /var/sonos/vendor/duncan3dc/domparser/src/Parser.php on line 35
PHP Fatal error:  Uncaught Error: Call to a member function getTag() on boolean in /var/sonos/vendor/duncan3dc/sonos/src/Speaker.php:66
Stack trace:
#0 /var/sonos/vendor/duncan3dc/sonos/src/Network.php(112): duncan3dc\Sonos\Speaker->__construct(Object(duncan3dc\Sonos\Devices\Device))
#1 /var/sonos/vendor/duncan3dc/sonos/src/Network.php(148): duncan3dc\Sonos\Network->getSpeakers()
#2 /var/sonos/vendor/duncan3dc/sonos/src/Network.php(212): duncan3dc\Sonos\Network->getSpeakerByRoom('Kitchen')
#3 /var/sonos/sonos.php(21): duncan3dc\Sonos\Network->getControllerByRoom('Kitchen')
#4 {main}
  thrown in /var/sonos/vendor/duncan3dc/sonos/src/Speaker.php on line 66

```

**Comment by duncan3dc** [comment]
Hi @tomsommer, thanks for your bug report.
Is this happening every time, or just in specific circumstances?
Are there are steps to reproduce this? (eg try to get the controller after moving the Kitchen to another group)

**Comment by tomsommer** [comment]
It happened after I added a stereo-pair of Ones, but I also updated to 9.8 at the same time, so hard to say.

**Comment by duncan3dc** [comment]
Thanks for the extra info, so has it stopped happening now? You're unable to reproduce?

**Comment by tomsommer** [comment]
Yes, related to #90

**Comment by duncan3dc** [comment]
Please can you try again with version 2.0.4 and provide a short script to reproduce the problem if it's still occurring with that new version

**Comment by tomsommer** [comment]
Works now. Awesome :)

**Comment by tomsommer** [comment]
2.0.4
```
PHP Warning:  DOMDocument::loadXML(): Empty string supplied as input in /var/sonos/vendor/duncan3dc/domparser/src/Parser.php on line 35
PHP Fatal error:  Uncaught Error: Call to a member function getTag() on boolean in /var/sonos/vendor/duncan3dc/sonos/src/Speaker.php:67
Stack trace:
#0 /var/sonos/vendor/duncan3dc/sonos/src/Network.php(113): duncan3dc\Sonos\Speaker->__construct(Object(duncan3dc\Sonos\Devices\Device))
#1 /var/sonos/sonos.php(20): duncan3dc\Sonos\Network->getSpeakers()
#2 {main}
  thrown in /var/sonos/vendor/duncan3dc/sonos/src/Speaker.php on line 67

```

**Comment by duncan3dc** [comment]
Thanks @tomsommer I'll try to reproduce again

**Comment by duncan3dc** [comment]
Thanks for bearing with me on this one, it should be fixed in [2.0.6](https://github.com/duncan3dc/sonos/releases/tag/2.0.6)
